marimo/_runtime/executor.py

```diff
@@ -65,9 +65,8 @@ async def execute_cell_async(
     graph: DirectedGraph,
     execution_type: str = "relaxed",
 ) -> Any:
-    return await EXECUTION_TYPES[execution_type].execute_cell_async(
-        cell, glbls, graph
-    )
+    executor = EXECUTION_TYPES[execution_type]()
```

Comment:
Unneeded churn

Reply:
Oh I see. Yeah, it's reasonable to put the change here.
But just setting last expr to None seems ideal. More execution types will require more churn and meta data ferrying.